Lib/fontTools/misc/psCharStrings.py

```diff
@@ -1043,8 +1043,7 @@ def setBytecode(self, bytecode):
 		self.program = None
 
 	def getToken(self, index,
-			len=len, byteord=byteord, basestring=basestring,
-			isinstance=isinstance):
+			len=len, byteord=byteord, isinstance=isinstance):
```

Comment:
Do any callers pass the `basestring`, would this change be an API break?

Reply:
Good catch.  My fault.  I added it back in #2242